inflammation/views.py

```diff
@@ -1,13 +1,16 @@
 """Module containing code for plotting inflammation data."""
+from typing import Dict
+import numpy as np
 from matplotlib import pyplot as plt
+from inflammation.models import Patient
 
 
-def visualize(data_dict):
+def visualize(data_dict: Dict[str, np.ndarray]):
     """Display plots of basic statistical properties of the inflammation data.
 
     :param data_dict: Dictionary of name -> data to plot
```

Comment:
@isabelle-gaffney @carlosroca-az @h4pZ Do we want to add a type hint at the parameter def as well?

Reply:
It is nice, that way your type checker and LSP can help when coding

Reply:
I personally really like these, and hopefully not too much work to add them throughout, but happy to be disagreed with!

Reply:
It is nice to have, if time allows

Reply:
ditto